modules/ROOT/pages/create-target.adoc

```diff
@@ -1,127 +1,140 @@
 +
-These credentials are used by the client application, {product-proxy}, and {product-automation} to read and write to your target database.
+These credentials are used by the client application and {product-proxy} to read and write to your target database.
+Make sure the token's role has sufficient permission to perform the actions required by your client application.
 
 . xref:astra-db-serverless:databases:secure-connect-bundle.adoc[Download your database's {scb}].
 The {scb-short} is a zip file that contains TLS encryption certificates and other metadata required to connect to your database.
+It is used during and after the migration process to securely connect to your {astra-db} database.
 +
 [IMPORTANT]
 ====
 The {scb-short} contains sensitive information that establishes a connection to your database, including key pairs and certificates.
 Treat it as you would any other sensitive values, such as passwords or tokens.
 ====
-+
-Your client application uses the {scb-short} to connect directly to {astra-db} near the end of the migration, and {cass-migrator} and {dsbulk-migrator} use the {scb-short} to migrate and validate data in {astra-db}.
 
-. Use `scp` to copy the {scb-short} to your client application instance:
+. Use your preferred method to copy the {scb-short} to your client application instance.
+For example, you could use `scp`:
 +
 [source,bash]
 ----
-scp -i <your_ssh_key> /path/to/scb.zip <linux user>@<public IP of client application instance>:
+scp -i some-key.pem /path/to/scb.zip user@client-ip-or-host:
 ----
 
 . Recreate your client application's schema on your {astra-db} database, including each keyspace and table that you want to migrate.
 +
 [IMPORTANT]
 ====
 On your new database, the keyspace names, table names, column names, data types, and primary keys must be identical to the schema on the origin cluster or the migration will fail.
+
+To help you prepare the schema from the DDL in your origin cluster, consider using the `generate-ddl` functionality in the {dsbulk-migrator-repo}[{dsbulk-migrator}].
 ====
 +
 Note the following limitations and exceptions for tables in {astra-db}:
 +
-* In {astra-db}, you must create keyspaces in the {astra-ui} or with the {devops-api} because xref:astra-db-serverless:cql:develop-with-cql.adoc[CQL for {astra-db}] doesn't support `CREATE KEYSPACE`.
-For instructions, see xref:astra-db-serverless:databases:manage-keyspaces.adoc[].
-* You can use {astra-db}'s built-in or standalone `cqlsh` to issue typical CQL statements to xref:astra-db-serverless:databases:manage-collections.adoc#create-a-table[create tables in {astra-db}].
+* In {astra-db}, you must xref:astra-db-serverless:databases:manage-keyspaces.adoc[create keyspaces in the {astra-ui} or with the {devops-api}] because xref:astra-db-serverless:cql:develop-with-cql.adoc[CQL for {astra-db}] doesn't support `CREATE KEYSPACE`.
```

Comment:
Not sure if you want to mention that the DataAPI clients work too (though they use the devops API): https://docs.datastax.com/en/astra-db-serverless/api-reference/admin-methods/create-keyspace.html

Reply:
With the migrations, they will not already be familiar with the Data API since they are coming from CQL/drivers. However, the keyspaces page will show them the Data API option. They will need to become familiar with the DevOps API anyways because of its role in Astra administration.